pkg/apis/mysql/v1alpha1/mysqlcluster_types.go

```diff
@@ -241,6 +245,17 @@ type VolumeSpec struct {
 	PersistentVolumeClaim *core.PersistentVolumeClaimSpec `json:"persistentVolumeClaim,omitempty"`
 }
 
+// ServiceSpec s the desired spec for addition configuration of MysqlCluster services
+type ServiceSpec struct {
+	// ServiceType defines a type for Services: ClusterIP, Loadbalancer, etc.
+	// +optional
+	ServiceType core.ServiceType `json:"type,omitempty"`
```

Comment:
If I need to use the `nodePort` Type, how do I specify `nodeport`

Reply:
Fair, actually, there are two ports opened: one for sidecar and another one for mysql.

The issue is even bigger, tho: you need separate nodePorts for replica and master